frontend/src/views/FinalSupplyEquipments/AddEditFinalSupplyEquipments.jsx

```diff
@@ -428,7 +430,6 @@ export const AddEditFinalSupplyEquipments = () => {
             gridOptions={gridOptions}
             loading={optionsLoading || equipmentsLoading}
             onCellEditingStopped={onCellEditingStopped}
-            stopEditingWhenCellsLoseFocus
```

Comment:
are we changing the way we save records?

Reply:
Good catch, this was in error. There is a ui issue with these date pickers that still needs to be resolved related to this param.

Reply:
I spent a couple hours trying to resolve this but there is a tricky propagation issue going on. For now FSE and Fuel Export will have this outside click issue. I have created a followup ticket to resolve this https://github.com/orgs/bcgov/projects/137/views/1?pane=issue&itemId=102406393&issue=bcgov%7Clcfs%7C2240